server/data/quotes.json

```diff
@@ -75,6 +75,10 @@
     "sentence": "Language shapes the way we think, and determines what we can think about.",
     "author": "Benjamin Lee Whorf"
   },
+  {
+  "sentence": "We pass at once into the magnificent edifice which bears the name of Panini as its architect and which justly commands the wonder and admiration of everyone who enters, and which, by the very fact of its sufficing for all the phenomenon which language presents, bespeaks at once the marvelous ingenuity of its inventor and his profound penetration of the entire material of the language.",
```

Comment:
The quote is too long, it might take too much attention off the main word as it will take up much space. Hope you can change this.

I'll be changing the guidelines as well to limit the number of characters for the quotes.

Reply:
@jayehernandez  let me modify the quote and request another pull request

Reply:
@jayehernandez  updated the quotes.json with a new quote and removed the lengthy quote.
Please go ahead and review

Reply:
thanks for changing it!